curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md

```diff
@@ -11,7 +11,7 @@ In this challenge you will have to create and save a record of a model.
 
 ## Instructions
 <section id='instructions'>
-Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must be conformant to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
+Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must be conformed to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

Comment:
```suggestion
Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must conform to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

Reply:
@thecodingaviator Why was the line I suggested (which is the same as yours) not able to be commit by user?  It looks like mine comment was resolved.

Reply:
Yes, I marked your comment as resolved. You had actually made a suggestion on the deleted line, i.e. the red line. If you unresolved the comment and try to click on Commit Suggestion, you’ll see what I mean

Reply:
Yes, I marked your comment as resolved. You had actually made a suggestion on the deleted line, i.e. the red line. If you unresolved the comment and try to click on Commit Suggestion, you’ll see what I mean

Reply:
That is odd.